docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -6,48 +6,80 @@ title: Schedule GPUs
 
 {% capture overview %}
 
-Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
-This page describes how users can consume GPUs and the current limitations.
+{% include feature-state-alpha.md %}
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes using the device plugins alpha feature.
+This page describes NVIDIA's supported method on how users can consume GPUs as well as the current limitations.
+
+Starting in version 1.8, Kubernetes provides a [device plugin framework](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)
+for vendors to advertise their resources to the kubelet without changing Kubernetes core code.
+
+Instead of writing custom Kubernetes code, vendors can implement a device plugin that can be deployed manually or as a DaemonSet.
+The targeted devices include GPUs, High-performance NICs, FPGAs, InfiniBand, and other similar computing resources that may require vendor specific initialization and setup.
 
 {% endcapture %}
 
 {% capture prerequisites %}
 
-1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise. Try to re-install Nvidia drivers if kubelet fails to expose Nvidia GPUs as part of Node Capacity. After installing the driver, run `nvidia-docker-plugin` to confirm that all drivers have been loaded.
-2. A special **alpha** feature gate `Accelerators` has to be set to true across the system: `--feature-gates="Accelerators=true"`.
+The list of prerequisites for using NVIDIA GPUs is described below:
+1. Kubernetes version needs to be greater than version 1.8
+2. Kubernetes nodes have to be pre-installed with NVIDIA drivers.
+3. Kubernetes nodes have to be pre-installed with [nvidia-docker 2.0](https://github.com/NVIDIA/nvidia-docker)
+4. The `DevicePlugins` feature gate enabled
+5. docker configured as the [default runtime](https://github.com/NVIDIA/nvidia-docker/wiki/Advanced-topics#default-runtime).
+6. NVIDIA drivers ~= 361.93
+
+Because this feature is still **alpha** you'll have to:
+1. Enable the feature gate `DevicePlugins` in Kubelet across GPU nodes: `--feature-gates="Accelerators=true"`.
 3. Nodes must be using `docker engine` as the container runtime.
+3. Nodes must be using the nvidia runtime as the docker [default runtime](https://github.com/NVIDIA/nvidia-docker/wiki/Advanced-topics#default-runtime).
```

Comment:
These 3 steps have been mentioned above?

Reply:
Thanks forgot to remove that